src/test/java/uk/gov/ons/ctp/response/action/endpoint/ActionPlanJobEndpointUnitTest.java

```diff
@@ -290,76 +283,4 @@ public void findActionPlanJob() throws Exception {
-    ResultActions actions = mockMvc.perform(postJson(String.format("/actionplans/%s/jobs", ACTIONPLANID),
-            ACTION_PLAN_JOB_INCORRECT_JSON));
-
-    actions.andExpect(status().isBadRequest())
-            .andExpect(handler().handlerType(ActionPlanJobEndpoint.class))
-            .andExpect(handler().methodName("executeActionPlan"))
-            .andExpect(jsonPath("$.error.code", is(CTPException.Fault.VALIDATION_FAILED.name())))
-            .andExpect(jsonPath("$.error.message", is(PROVIDED_JSON_INCORRECT)))
-            .andExpect(jsonPath("$.error.timestamp", isA(String.class)));
-  }*/
-
-  /**
-   * A Test to execute an action plan providing json failing validation
-   *
-   * @throws Exception exception thrown when postJson does
-   */
-  @Test
-  public void executeActionPlanJsonFailingValidation() throws Exception {
-    final ResultActions actions =
-        mockMvc.perform(
-            postJson(
-                String.format("/actionplans/%s/jobs", ACTIONPLANID),
-                ACTION_PLAN_JOB_JSON_FAILING_VALIDATION));
-
-    actions
-        .andExpect(status().isBadRequest())
-        .andExpect(handler().handlerType(ActionPlanJobEndpoint.class))
-        .andExpect(handler().methodName("executeActionPlan"))
-        .andExpect(jsonPath("$.error.code", is(CTPException.Fault.VALIDATION_FAILED.name())))
-        .andExpect(jsonPath("$.error.message", is(INVALID_JSON)))
-        .andExpect(jsonPath("$.error.timestamp", isA(String.class)));
-  }
-
-  /**
-   * A Test to execute an action plan providing good json
-   *
-   * @throws Exception exception thrown when postJson does
-   */
-  @Test
-  public void executeActionPlanGoodJsonProvided() throws Exception {
-    when(actionPlanJobService.createAndExecuteActionPlanJob(any(ActionPlanJob.class)))
-        .thenReturn(actionPlanJobs.get(0));
-    when(actionPlanService.findActionPlanById(ACTIONPLANID)).thenReturn(actionPlans.get(0));
-
-    final ResultActions actions =
-        mockMvc.perform(
-            postJson(String.format("/actionplans/%s/jobs", ACTIONPLANID), ACTION_PLAN_JOB_JSON));
-
-    actions
-        .andExpect(status().isCreated())
-        .andExpect(handler().handlerType(ActionPlanJobEndpoint.class))
-        .andExpect(handler().methodName("executeActionPlan"))
-        .andExpect(jsonPath("$.*", hasSize(6)))
-        .andExpect(jsonPath("$.id", is(ACTION_PLAN_JOB_ID_1.toString())))
-        .andExpect(jsonPath("$.actionPlanId", is(ACTIONPLANID.toString())))
-        .andExpect(jsonPath("$.createdBy", is(CREATED_BY_SYSTEM)))
-        .andExpect(jsonPath("$.state", is(ActionPlanJobDTO.ActionPlanJobState.COMPLETED.name())))
-        .andExpect(
-            jsonPath(
-                "$.createdDateTime", is(new DateMatcher(CREATED_DATE_TIME_ACTION_PLAN_JOB_ID_1))))
-        .andExpect(
-            jsonPath(
-                "$.updatedDateTime", is(new DateMatcher(UPDATED_DATE_TIME_ACTION_PLAN_JOB_ID_1))));
-  }
```

Comment:
These look like valid tests to me

Reply:
No longer accepting any JSON input for this endpoint